### PR TITLE
Add alerting policy templates for GCE Instances Observability tab.

### PR DESCRIPTION
**Pull request by williamang2**
Screenshot 2022-12-12 3 31 44 PM
Ran gcloud command to verify template validity.

### CODE REVIEW

File: alerts/google-gce-instances/memory-utilization-too-high.v1.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            "crossSeriesReducer": "REDUCE_NONE",
+            "perSeriesAligner": "ALIGN_MEAN"
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": "0s",
+        "trigger": {
+          "count": 1
+        },
+        "thresholdValue": 100
```
Comment (draffensperger) [inline alerts/google-gce-instances/memory-utilization-too-high.v1.json:21]: How did you pick a threshold of `100` here? I would have expected something more like 80% or 90%

Comment (williamang2) [reply alerts/google-gce-instances/memory-utilization-too-high.v1.json]: Per chat, reducing this threshold to 90%.

**Review by huygaa11 — COMMENTED** [review]
should the folder be named "google-gce"?

**Comment by huygaa11** [comment]
> Screenshot 2022-12-12 3 31 44 PM Ran gcloud command to verify template validity.

Probably the screenshot of the created alert in Pantheon UI.

**Comment by williamang2** [comment]
> > Screenshot 2022-12-12 3 31 44 PM Ran gcloud command to verify template validity.
> 
> Probably mean the screenshot of the created alert in Pantheon UI.

The integrations doc was a little unclear on this, so I looked at the past merged PRs and it looked like the command line output was what they were looking for (https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/400, https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/377). I can leave a comment in the integrations doc to confirm what the preference for the confirmation is.

**Comment by williamang2** [comment]
> should the folder be named "google-gce"?

You're right, I forgot that was where we landed on the naming schema. Folder is now renamed to google-gce.

File: alerts/google-gce/cpu-utilization-too-high.v1.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "displayName": "CPU Utilization",
```
Comment (cocosheng) [inline alerts/google-gce/cpu-utilization-too-high.v1.json:2]: Please match displayName with the ones in metadata.yaml, same for other policies.

File: alerts/google-gce/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+alert_policy_templates:
+-
+  id: cpu-utilization-too-high
+  display_name: cpu utilization too high
```
Comment (cocosheng) [inline alerts/google-gce/metadata.yaml:4]: please match with the displayName in actual policies. Apply to other ones.

**Review by cocosheng — CHANGES_REQUESTED** [review]

File: alerts/google-gce/metadata.yaml (hunk 3 above)
Comment (williamang2) [reply alerts/google-gce/metadata.yaml]: Display names in metadata.yaml now matches the display names in the individual json files.

File: alerts/google-gce/cpu-utilization-too-high.v1.json (hunk 2 above)
Comment (williamang2) [reply alerts/google-gce/cpu-utilization-too-high.v1.json:2]: Display names in json files now match metadata.yaml.

**Review by cocosheng — APPROVED** [review]

**Pull request merged**